pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,27 @@
+# Getting Started
+
+Integrating Pyth price feeds is quick and easy. Pyth price feeds are permissionless and available on-chain. You **don't** need to sign up or request an API key
+
+Pyth offers several different resources to help you get started.
+The Build section provides resources for developers integrating Pyth price feeds into their applications.
+The Learn section provides general meterial for anyone interested in understanding how the protocol works.
+
+## Build
+
+Developers interested in using Pyth can refer to the following resources:
+
+- [Create-your-first-pyth-app](./create-your-first-pyth-app/) is a tutorial that walks the reader through all of the steps required to develop, test and deploy a contract using Pyth price feeds. This guide is tailored toward new developers with less contract development experience.
+- [User-real-time-data](./use-real-time-data/) is a how-to guide that provides the minimal steps to integrate price feeds into your app. This guide is targeted towards more experienced developers who know the basics of smart contract development.
+
+In addition to the two guides above, the following resources will be useful for developers as they integrate:
+
+- [API-reference](./api-reference.mdx) provides a detailed overview of Pyth's on-chain methods.
+- [Price Feed Ids](./price-feed-ids.mdx) list the price feed ids for all the assets supported by Pyth.
+- [Contract-addresses](./contract-addresses/) provides the contract addresses for Pyth on different chains.
+- [Best-practices](./best-practices.mdx) to use Pyth price feeds safely and effectively in your application.
```

Comment:
All of the items in a bulleted list need to have the same grammatical structure. In this case, all of the items up to here are "[page] (present tense verb)" (which is actually a great structure IMO). The last item should follow that: "Best practices explains how to use ..."